community/emacs/build

```diff
@@ -1,8 +1,6 @@
 #!/bin/sh -e
 
 ./configure \
-    --prefix=/usr \
-    --with-gnutls=no
+    --prefix=/usr
 
-make
 make DESTDIR="$1" install
```

Comment:
This file is from the regular `emacs` package. :+1:

Reply:
errg

Reply:
thank you for teaching me btw

Reply:
No problem. :)